### PR TITLE
Upgrade brainzutils and sentry-sdk

### PR DESCRIPTION
**Pull request by amCap1712**
Apart from upgrading brainzutils and sentry-sdk, I have also removed configuring email logging. We do not have the required configurations anyways and never use it. I have also removed a loggers.py which does not seem to be used anywhere.

### CODE REVIEW

File: metabrainz/__init__.py [hunk 1]
```diff
@@ -74,7 +74,6 @@ def create_app(debug=None, config_path = None):
 
     app.init_loggers(
         file_config=app.config.get('LOG_FILE'),
-        email_config=app.config.get('LOG_EMAIL'),
```
Comment (alastair) [inline metabrainz/__init__.py:77]: we should also remove this from the consul configs in docker-server-configs

File: metabrainz/model/payment.py [hunk 2]
```diff
@@ -170,9 +170,7 @@ def process_paypal_ipn(cls, form):
         if form['payment_status'] != 'Completed':
             # TODO(roman): Convert to regular `logging.info` call when such detailed logs
             # are no longer necessary to capture.
-            get_sentry_client().captureMessage("PayPal: Payment is not completed",
-                                               level=logging.INFO,
-                                               extra={"ipn_content": form})
+            capture_message("PayPal: Payment is not completed", level="info", extra={"ipn_content": form})
```
Comment (alastair) [inline metabrainz/model/payment.py:173]: just confirming, have you checked that this message gets reported properly?

Comment (amCap1712) [reply metabrainz/model/payment.py:173]: Yes.

**Review by alastair — APPROVED** [review]
Based on our changes yesterday this looks fine to me, though I didn't test it myself.

**Pull request merged**